pages/i18n/localization.json

```diff
@@ -164,10 +154,10 @@
             "heading_libraries": "Публічні бібліотеки"
           },
           "cards_texts": {
-            "text_for_cases": "Для випадків, коли ви не знайшли потрібну готову інтеграцію ",
-            "text_integrate": "Інтегруйте <a href=\"/\" target=\"_blank\" class=\"unlimited-integrations-link-style\">знайомі служби</a> кількома кліками",
-            "text_get_data": "Отримуйте дані про будь-які івенти у CRM та автоматизуйте складні бізнес-процеси",
-            "text_for_custom": "Для кастомних інтеграцій зі специфічними продуктами"
+            "text_1": "Для випадків, коли ви не знайшли потрібну готову інтеграцію ",
```

Comment:
why do you use these names? its difficult to read and understand what is it 
https://dook.pro/blog/programming/18-keep-it-clean-coding-good-practices-naming